packages/lit-element/src/test/polyfill-support/lit-element-ce-sd-noPatch_test.html

```diff
@@ -32,7 +32,7 @@
         mocha,
         sessionFinished,
         sessionFailed,
-      } from '../../../../tests/node_modules/@web/test-runner-mocha/dist/standalone.js';
+      } from '../../../../../node_modules/@web/test-runner-mocha/dist/standalone.js';
       // import { mocha, sessionFinished, sessionFailed } from '@web/test-runner-mocha';
```

Comment:
I tried switching to the non-relative import and got the following logged in the dev mode run
```
 🚧 Browser logs on Webkit:
      An error was thrown in a Promise outside a test. Did you forget to await a function or assertion?
      Error: test error
        at render (../lit-element/src/test/lit-element_test.ts:257:26)
        at update (../lit-element/src/lit-element.ts:160:29)
        at performUpdate (../reactive-element/src/reactive-element.ts:1303:20)
        at ../reactive-element/src/reactive-element.ts:1207:38
```
But the tests passed. 🤷 Probably not worth trying to investigate for now.

Reply:
On this line in the HTML test? This is a little bit of a different context to the relative/self-reference change in this PR. To test that the dev/prod check is working, you'll want to change an import in one of the `src/tests/*.ts` files that is importing a library from `../something.js`

Reply:
Oh right. This is definitely unrelated to changes in this PR since it's not a self reference, nor is this a ts file. 🤦 I just saw the comment above saying node resolution didn't seem to work there and thought I'd give it a whirl. Please disregard!